### ISSUE TITLE
Train from Scratch of ViG

### ISSUE THREAD

**Issue by hsama95** [huawei-noah/Efficient-AI-Backbones#136]
Hi, 

Thanks for your great work and sharing the codes, I am trying to reproduce the results by training the ViG from scratch. And I followed the README file to install required packages with specific version number, but I got the following training status, it looks a little weird. Could you please check it and let me know whether it is normal?

Thanks!

<img width="1114" alt="Screen Shot 2022-09-11 at 2 39 37 PM" src="https://user-images.githubusercontent.com/43621476/189550013-fcc29191-b909-416d-a6a2-e1c7264f85b8.png">

**Comment by iamhankai** [comment]
From your log, the training loss is decreasing normally, but the val loss and acc are not normal. Please check the val data or code.

Here is my log:

![截图](https://user-images.githubusercontent.com/9500784/189850756-455b913e-8b24-4729-9849-94c9f42179fd.PNG)

**Comment by hsama95** [comment]
Hi, thanks for your reply, I downloaded the IamgeNet dataset from Kaggle, https://www.kaggle.com/competitions/imagenet-object-localization-challenge/data, and do you make some modification about the dataset?

**Comment by iamhankai** [comment]
We followed the [official pytorch example](https://github.com/pytorch/examples/tree/main/imagenet): move and extract the training and validation images to labeled subfolders, using [the following shell script](https://github.com/pytorch/examples/blob/main/imagenet/extract_ILSVRC.sh)
